admin-manual/maintenance/maintenance.rst

```diff
@@ -160,11 +160,11 @@ Data to back up from an Archivematica instance:
 
 #. MCP database (see below for details)
 #. SS database (see below for details)
-#. Elasticsearch index (see below for details)
+#. Elasticsearch index/indices (see below for details)
```

Comment:
Sorry I was not clear enough. I meant that we should use "indexes" or "indices" instead of "index". Just "indexes" is probably better after reading the entire doc ;)

Reply:
Oh I see. I changed it!